lib/dts-lib.robot

```diff
@@ -2,18 +2,54 @@
 Resource    terminal.robot
 
 
+*** Variables ***
+# Default DTS link for iPXE boot, can be overwritten by CMD:
+${DTS_IPXE_LINK}=               http://boot.dasharo.com/dts/dts.ipxe
+${BOOT_DTS_FROM_IPXE_SHELL}=    ${FALSE}
```

Comment:
Shouldn't this be defined in `platform-configs/include/default.robot`?

Reply:
Agreed, changed [here](https://github.com/Dasharo/open-source-firmware-validation/compare/be03f347bc525cf96d2df3062466fac08489a04f..18a702db97333836ce2084ea95643c7a74e78e78).